content/pages/comparisons/gitbook.html

```diff
@@ -42,6 +42,7 @@
     {{ comparison.feature_row("Redirects", "Redirect users to the right page", "fa-arrows-turn-right", on_other=True) }}
     {{ comparison.feature_row("Translations", "Serve translated versions of your docs", "fa-language", on_other=True) }}
     {{ comparison.feature_row("Private documention", "Give access only to trusted people", "fa-lock", on_other=True) }}
+    {{ comparison.feature_row("Deploy previews from forked repositories", "Visualize edits from external collaborators", "fa-people-group") }}
```

Comment:
"Deploy" suggests that forked repositories can change production documentation, so sounds a bit wrong. I'd stick to using "preview" for pull request builds

```suggestion
    {{ comparison.feature_row("Preview changes from forked repositories", "Visualize edits from external collaborators", "fa-people-group") }}
```